app/dashboard/templates/bounty_details.html

```diff
@@ -20,77 +20,102 @@
         <div class="col body">
             <div class="container-fluid" id="bounty_detail">
             {% include 'shared/bounty_def.html' %}
+            </div>
+
+            <div class="alert info transaction-status" style="display:none;">
+                <strong style="font-size: 22px;">Your transaction has been posted to web3</strong>
+                <br>
+                This funded issue has recently been updated and while the blockchain syncs it has
+                <a target="_blank" href="javascript:void(0)" id="pending_changes">pending changes</a>.
+                <br>
+                Please wait a minute or two for web3 to sync
+                <a target="_blank" href="javascript:void(0)" id="transaction_url">this transaction</a>.
+                <br>
+                (Please DO NOT close the browser tab. This page will automatically refresh as soon as the blockchain is updated.)
+            </div>
+
+            <div class="row loading justify-content-center align-items-center loading_img">
+                <img src='/static/v2/images/loading_v2.gif'>
+            </div>
 
-                <div class="row loading justify-content-center align-items-center loading_img">
-                    <img src='/static/v2/images/loading_v2.gif'>
+            <div class="row nonefound" style="display:none;">
+                <img src="/static/v2/images/shrug_bot.png">
+                <h3>No funded issue found.</h3>
+                <p>Please go back to the <a href="/explorer">Issue Explorer</a> and try again.</p>
+                <br>
+                <h6>Be the OSS Funding you wish to see in the world.</h6>
+                <p id='submitsolicitation'>Looking to fund some work?  You can submit a new Funded Issue <a href="/funding/new">here</a>.</p>
+            </div>
+
+            <div class="row ml-2 mr-2 pl-4 pr-4 waiting_room_entertainment" style="display: none;">
+                {% include 'shared/rain.html' with class="color" %}
+                <div class="quote-container">
+                    <div class="quote_text"></div>
+                    <div class="quote_author"></div>
                 </div>
-                <div class="row nonefound" style="display:none;">
-                    <img src="/static/v2/images/shrug_bot.png">
-                    <h3>No funded issue found.</h3>
-                    <p>Please go back to the <a href="/explorer">Issue Explorer</a> and try again.</p>
-                    <br>
-                    <h6>Be the OSS Funding you wish to see in the world.</h6>
-                    <p id='submitsolicitation'>Looking to fund some work?  You can submit a new Funded Issue <a href="/funding/new">here</a>.</p>
+            </div>
+
```

Comment:
are these all just indentation changes?

Reply:
Not all of them are indentation changes. There are some additional HTML DIV sections and CSS classes to get the Design match that of the spec